lib/matplotlib/patches.py

```diff
@@ -3710,23 +3727,25 @@ def __init__(self,
                      widthA=1., angleA=None,
                      widthB=1., angleB=None):
             """
-            *widthA*
-              width of the bracket
+            Parameters
+            ----------
+            widthA : float, optional, default : 1.0
+                Width of the bracket
 
-            *lengthA*
-              length of the bracket
+            lengthA : int
```

Comment:
I'm slightly confused, `__init__` just above this docstring doesn't take a `lengthA` or `lengthB` argument, is the docstring just wrong here at the moment?